lua.go

```diff
@@ -437,6 +437,14 @@ func (l *State) Load(r io.Reader, chunkName string, mode string) error {
 	return nil
 }
 
+func (l *State) Dump(w io.Writer) error {
```

Comment:
This needs a comment. I generally use language similar to the corresponding entry in the Lua manual, and include a link to the original docs. See the comment for `Load` above.